service/src/main/java/com/commercetools/pspadapter/payone/domain/ctp/paymentmethods/PaymentMethod.java

```diff
@@ -63,6 +63,16 @@
      */
     INVOICE_KLARNA(MethodKeys.INVOICE_KLARNA);
 
+    /**
+     * @see MethodKeys#BANK_TRANSFER_IDEAL
+     */
+//    BANK_TRANSFER_IDEAL(MethodKeys.BANK_TRANSFER_IDEAL),
```

Comment:
Why you need to comment it out?

Reply:
Because in our existing tests, we are loading all the payment types and filling mapping those with relevant factory classes. As we still do not have factory classes created, tests are failing in the build process. So, for now I have commented them out.

Reply:
in this line -> https://github.com/commercetools/commercetools-payone-integration/pull/300/files/f0be6df591133781904201787d303e9b27dde16e#diff-3439e6d93c20fc6f9d51971b99d0cd1eR80